src/main/java/ca/on/oicr/pineryreports/reports/impl/PreciseCaseReport.java

```diff
@@ -151,6 +155,9 @@ protected void collectData(PineryClient pinery) throws HttpResponseException, IO
                 allPreciseSamplesByIdentity
                     .get(identity.getId()) // Get children of this identity
                     .stream()
+                    .filter(s -> 06 != SampleUtils.getTubeNumber(s.getName())) // See GLT-3314
+                    .filter(s -> 17 != SampleUtils.getTubeNumber(s.getName()))
+                    .filter(s -> 31 != SampleUtils.getTubeNumber(s.getName()))
```

Comment:
Add a static `Predicate` that excludes based on all 3 tube numbers and reference that in all 4 places this is done, so the tube numbers are defined in a single place. While it probably won't do any harm in this instance, the `6` is specified in octal and shouldn't be

Reply:
Fixed one and forgot to fix the others, so putting them all in one place is probably a good idea

Reply:
Done